### DISCUSSION TITLE
Migrate to native named services

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno.extensions#2384] [Q&A]
Are we going to migrate to .NET 8 native named services in DI?
@nickrandolph @jeromelaban

**Comment by dansiegel** [comment]
From an Extensions perspective what is it you feel should be using named Services? Keep in mind that as long as you're using .NET 8+ Named Services are available for you in your own code while you're using Extensions.

**Reply by xperiandri** [reply]
Yep, I mean Uno.Extensions source code migration

**Comment by DevTKSS** [comment]
failing on Named Services + Named Options also here #2769
